src/BonsaiBalance.Mobile/BaseProject/.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.5
```

Comment:
Ruby?

Reply:
Ruby is for cocoapods, which is for iOS stuff